src/plugin.cpp

```diff
@@ -149,8 +149,10 @@ void registerModuleChangeSubscriptions(sr::Session& sess, ietf::sys::PluginConte
 {
     const auto change_callbacks = {
         ModuleChangeCallback { "/ietf-system:system/hostname", ietf::sys::sub::change::HostnameModuleChangeCb(ctx.getModuleChangeContext()) },
-        ModuleChangeCallback {
-            "/ietf-system:system/clock/timezone-name", ietf::sys::sub::change::ClockTimezoneNameModuleChangeCb(ctx.getModuleChangeContext()) },
+        ModuleChangeCallback { "/ietf-system:system/clock/timezone-name", ietf::sys::sub::change::ClockTimezoneNameModuleChangeCb(ctx.getModuleChangeContext()) },
+        ModuleChangeCallback { "/ietf-system:system/dns-resolver/search", ietf::sys::sub::change::DnsSearchModuleChangeCb(ctx.getModuleChangeContext()) },
+        ModuleChangeCallback { "/ietf-system:system/dns-resolver/server//*", ietf::sys::sub::change::DnsServerModuleChangeCb(ctx.getModuleChangeContext()) },
```

Comment:
`/ietf-system:system/dns-resolver/server` path should work fine. No need for `//*`.

Reply:
Oh right, my bad. `//*` includes all nested elements. It can stay like this.